src/status_im2/contexts/wallet/common/sheets/network_preferences/style.cljs

```diff
@@ -1,4 +1,13 @@
-(ns status-im2.contexts.wallet.common.sheets.network-preferences.style)
+(ns status-im2.contexts.wallet.common.sheets.network-preferences.style
+  (:require [quo.foundations.colors :as colors]))
+
+(def blur
+  {:position      :absolute
+   :top           0
+   :left          0
+   :right         0
+   :bottom        0
+   :overlay-color colors/neutral-80-opa-80-blur})
```

Comment:
Afaik if you look at the designs this is using the overlay component. 
@mohsen-ghafouri recently added it. Can we use this instead? 
https://www.figma.com/file/WQZcp6S0EnzxdTL4taoKDv/Design-System-for-Mobile?type=design&node-id=5410-137985&mode=design&t=RY24LelClKkTnsEg-0

<img width="321" alt="Screenshot 2023-12-18 at 13 28 20" src="https://github.com/status-im/status-mobile/assets/22799766/f8f975b7-5926-4770-88db-18cb62398e9f">

Reply:
yes, I fully tested with type blur, but it also has type `:shell`